index.js

```diff
@@ -325,6 +325,15 @@ function handleSpawned(spawned, context) {
 	});
 }
 
+const setupTimeout = function (spawned, {timeout, killSignal}, context) {
```

Comment:
```suggestion
const setupTimeout = (spawned, {timeout, killSignal}, context) => {
```

Reply:
I prefer using `const` and arrow functions for everything.

Reply:
Alright I will also send a follow-up PR to do that on all functions in the repository.